jsonschema/cli.py

```diff
@@ -69,19 +70,49 @@ def validation_success(self, **kwargs):
 
 @attr.s
 class _PrettyFormatter(object):
+    _MESSAGE_BAR_CHAR = '═'
+    _MESSAGE_CORNER_CHARS = ('╒', '╕')
+    _MESSAGE_FORMAT = '{}══[{}]═══({})'
+    _MESSAGE_MAX_LENGTH = 79
+
+    @classmethod
+    def _json_formatter(cls, x):
```

Comment:
This doesn't necessarily seem like it needs to be a classmethod -- any reason it shouldn't just be a regular function?

Also, rather than adding newlines to the separators, probably instead we just want `indent=True`?

Reply:
I could swear I replied about `@classmethod` earlier, but I don't see it here now.

If the method isn't annotated as `@classmethod` or `@staticmethod`, the tests fail.  If I use `@staticmethod`, the style tests tell me that methods should never be static, so I guess it needs to be `@classmethod`.

I agree about the JSON formatting.  I think I'd done it that way to make the `json.dumps()` output somewhat match the `repr` output that was used before.  The `indent` argument takes a number of spaces to indent, so I'll specify 4, to match the Python indentation style.  (JSON doesn't have an official standard indent size.)

Reply:
Yes but why is it on the class at all, instead of just being a function?